include/globals.h

```diff
@@ -143,6 +143,7 @@
 // C Helpers and Macros
 
 #define NAME_OF(x)          #x
+#define ACTUAL_NAME_OF(x)   ((#x) + 1)
```

Comment:
That's fair feedback, which also points out that the macro name is not ideally chosen. It does what NAME_OF does, but just skips the first character; it will yield the actual property name in case member variables are prefixed with an underscore. I just couldn't get myself to name it something like `NAME_OF_WITHOUT_FIRST_CHARACTER`.

Anyway, I just noticed that VS Code shows the comment with the symbol when one hovers the mouse over the latter, if the comment immediately precedes it. So, that's what I'll do.